### ISSUE TITLE
CORS issue on album arts

### ISSUE THREAD

**Issue by asa767** [AzuraCast/AzuraCast#5763]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

0.17.3

### What happened?

Getting now playing data from live service (Websocket/EventSource) is OK. But when my web application tries to get the album image I get the error below:

Access to XMLHttpRequest at 'https://admin.medreseradyo.com/api/station/1/art/4d884b096e73f0893fa165b4-1658576795.jpg' from origin 'https://test.medreseradyo.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by ogrenci01** [comment]
Hi @asa767 ;

Merhaba;

Hello;

In English: Go to the System Settings page from the admin panel. From the Security and Privacy tab, put the "*" (star) sign in the "Access-Control-Allow-Origin" section and save. Try again later.

In Turkish: Yönetici panelinden Sistem Ayarları sayfasına gidin. Güvenlik ve Gizlilik sekmesinden "Access-Control-Allow-Origin" bölümüne "*" (yıldız) işaretini koyup kaydedin. Daha sonra tekrar deneyin.

**Comment by asa767** [comment]
> Hi @asa767 ;
> 
> In English: Go to the System Settings page from the admin panel. From the Security and Privacy tab, put the "*" (star) sign in the "Access-Control-Allow-Origin" section and save. Try again later.
> 
> In Turkish: Yönetici panelinden Sistem Ayarları sayfasına gidin. Güvenlik ve Gizlilik sekmesinden "Access-Control-Allow-Origin" bölümüne "*" (yıldız) işaretini koyup kaydedin. Daha sonra tekrar deneyin.

I did. Nothing changed.  I also rebooted all system after this setting.  Still getting the error.

**Comment by ogrenci01** [comment]
Hi;

In English: If you are using Firewall on the servers you own, add the IP addresses to the Whitelist mutually. If you're using Cloudflare, allow IP addresses. Azuracast API services are working normally at the addresses you sent.

In Turkish: Sahip olduğunuz sunucularda Güvenlik Duvarı kullanıyorsanız IP adreslerini karşılıklı olarak Beyaz Listeye ekleyin. Cloudflare kullanıyorsanız IP adreslerine izin verin. Gönderdiğiniz adreslerde Azuracast API servisleri olağan şekilde çalışmaktadır.

**Comment by asa767** [comment]
I think It's not a firewall problem. If you check test.medreseradyo.com by clicking the radio icon from a CORS disabled browser It works normally. You may use https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf add on for this purpose.

**Comment by asa767** [comment]
Here are some screenshots about requests

It's OK when connecting to mp3 stream. It returns the needed "Access-Control-Allow-Origin" header.
<img width="684" alt="Ekran Resmi 2022-10-01 15 53 44" src="https://user-images.githubusercontent.com/26309435/193410574-abd7a968-83d3-4798-b079-914a90d26c02.png">

For album art there is no header for "Access-Control-Allow-Origin".
<img width="791" alt="Ekran Resmi 2022-10-01 15 54 03" src="https://user-images.githubusercontent.com/26309435/193410579-3d538230-b3c5-47c4-a0a8-c6b3889fbda9.png">

**Comment by BusterNeece** [comment]
After looking into this, I was able to identify the problem: if your filesystem is local, we used nginx's `X-Accel-Redirect` functionality to speed up art delivery; it turns out, however, that in using this functionality, headers from the original response are stripped and replaced by the ones nginx would serve for that file by default. That also removed our `Access-Control-Allow-Origin` header in the process.

This has been fixed in the latest update to the Rolling Release branch for all artwork-related API endpoints.
